_documentation/control-focus-modes.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Control Focus Modes
```

Comment:
Is this page necessary / accurate?  Because you can't actually control your Focus Modes with FocusCuts?  You can only get what Focus Modes you have right?  So, to me at least, it seems odd to have a page about controlling Focus Modes when it's not a feature we actually offer.

Maybe it should sit under the shortcut actions in an 'Example Shortcuts' section?

Reply:
I have a feeling it's a question we're going to get a lot, hence the section and suggestion to use Shortcuts. We could add a leading sentence that it can change the focus mode directly.

Reply:
That's fair, I may come back and adjust the wording later, but that's a future project.